### PR TITLE
migrate react docs to v10

### PR DESCRIPTION
**Pull request by bautistaaa**
theres some unknowns here so will mark places where I'm not very confident what the api looks like
part of #2115

### CODE REVIEW

File: www/docs/reactjs/useInfiniteQuery.md [hunk 1]
```diff
@@ -90,7 +89,7 @@ import { trpc } from '../utils/trpc';
 export function MyComponent() {
   const utils = trpc.useContext();
 
-  const myMutation = trpc.useMutation('infinitePosts.add', {
+  const myMutation = trpc.proxy.infinitePosts.add.useMutation({
     onMutate({ post }) {
       await utils.cancelQuery(['infinitePosts']);
```
Comment (bautistaaa) [inline www/docs/reactjs/useInfiniteQuery.md:94]: unsure if `utils` requires any changes like `utils.proxy.cancelQuery()`

File: www/docs/reactjs/useInfiniteQuery.md [hunk 2]
```diff
@@ -90,7 +89,7 @@ import { trpc } from '../utils/trpc';
...
       const allPosts = utils.getInfiniteQueryData(['infinitePosts', { limit: 10 }]);
```
Comment (bautistaaa) [inline www/docs/reactjs/useInfiniteQuery.md:95]: unsure if utils requires any changes like `utils.proxy.infinitePosts.cancelQuery({ limit: 10 })`

File: www/docs/reactjs/useInfiniteQuery.md [hunk 3]
```diff
@@ -90,7 +89,7 @@ import { trpc } from '../utils/trpc';
 export function MyComponent() {
   const utils = trpc.useContext();
```
Comment (bautistaaa) [inline www/docs/reactjs/useInfiniteQuery.md:90]: proxy?

File: www/docs/reactjs/invalidateQueries.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ import { trpc } from '../utils/trpc'
 // In component:
 const utils = trpc.useContext();
```
Comment (bautistaaa) [inline www/docs/reactjs/invalidateQueries.md:20]: does this need `proxy`?

File: www/docs/reactjs/useInfiniteQuery.md (hunk 1 above)
Comment (sachinraja) [reply www/docs/reactjs/useInfiniteQuery.md:94]: That's the plan but it hasn't been implemented yet (see https://github.com/trpc/trpc/issues/2113) so it's ok to leave this as is

**Review by sachinraja — APPROVED** [review]

**Review by KATT — APPROVED** [review]
🐐 🐐 🐐 🐐 🐐

**Comment by KATT** [comment]
@all-contributors add @bautistaaa for docs, examples

**Pull request merged**